app/views/items/_item_grid.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="items">
+  <div class="container">
+      <div class="row ">
+        <% @items.each do |item| %>
```

Comment:
- style: the item grid should not be dependent on a proper global variable (`@items`) to be set; instead you can pass arguments through the `render` helper, which should be used to include a common API; this makes it possible to use the item grid multiple times on one page or in different contexts and is therefore IMO preferable.